core/vm/actor/builtin/v2/market/market_actor.hpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "vm/actor/builtin/v0/market/market_actor.hpp"
+
+namespace fc::vm::actor::builtin::v2::market {
+  using primitives::DealWeight;
+
+  using Construct = v0::market::Construct;
+  using AddBalance = v0::market::AddBalance;
+  using WithdrawBalance = v0::market::WithdrawBalance;
+  using PublishStorageDeals = v0::market::PublishStorageDeals;
```

Comment:
```suggestion
  using v0::market::AddBalance;
  using v0::market::Construct;
  using v0::market::PublishStorageDeals;
  using v0::market::WithdrawBalance;
```

Reply:
I want to use
`using Construct = v2::market::Construct` etc. in v3